.github/workflows/validate_internal_docs_links.yml

```
@@ -0,0 +1,23 @@
+name: Validate internal documentation links
+
+"on": [pull_request]
+
+jobs:
+    tests:
+        name: mkdocs serve strict
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4.0.0
+
+            - name: Set up Python
+              uses: actions/setup-python@v4.7.0
```

```suggestion
              uses: actions/setup-python@v4
```
Same here; really doubt we have anything that would break with minor version changes.